src/Atc.Rest.MinimalApi/Filters/Endpoints/ValidationFilter.cs

```diff
@@ -29,21 +29,9 @@ public class ValidationFilter<T> : IEndpointFilter
 
     private static Dictionary<string, string[]> ValidateUsingDataAnnotations(
         object objectToValidate)
-    {
-        var result = new Dictionary<string, string[]>(StringComparer.Ordinal);
-
-        if (MiniValidator.TryValidate(objectToValidate, out var errors))
-        {
-            return result;
-        }
-
-        foreach (var error in errors)
-        {
-            result.Add(error.Key, error.Value);
-        }
-
-        return result;
-    }
+        => MiniValidator.TryValidate(objectToValidate, out var errors)
```

Comment:
Wait... are you sure that `TryValidate()` returns `true` if there are no errors? That sounds a bit weird to me since all other `TrySomething()` methods will return `true` if the "try" operation succeeds, and returns false if it some how ends up with an exception.

Maybe we should double check this and add another check if `errors` is empty

Something like this:
```cs
MiniValidator.TryValidate(objectToValidate, out var errors) && errors.Any()
```

Reply:
Yup.. taken from the implementation:

>     /// <summary>
>     /// Determines whether the specific object is valid.
>     /// </summary>
>     /// <typeparam name="TTarget"></typeparam>
>     /// <param name="target">The object to validate.</param>
>     /// <param name="serviceProvider">The service provider to use when creating ValidationContext.</param>
>     /// <param name="recurse"><c>true</c> to recursively validate descendant objects; if <c>false</c> only simple values directly on <paramref name="target"/> are validated.</param>
>     /// <param name="allowAsync"><c>true</c> to allow asynchronous validation if an object in the graph requires it.</param>
>     /// <param name="errors">A dictionary that contains details of each failed validation.</param>
>     /// <returns><c>true</c> if <paramref name="target"/> is valid; otherwise <c>false</c>.</returns>
>     /// <exception cref="ArgumentNullException">Thrown when <paramref name="target"/> is <c>null</c>.</exception>
>     /// <exception cref="ArgumentException">Throw when <paramref name="target"/> requires async validation and <paramref name="allowAsync"/> is <c>false</c>.</exception>
>     private static bool TryValidateImpl<TTarget>(TTarget target, IServiceProvider? serviceProvider, bool recurse, bool allowAsync, out IDictionary<string, string[]> errors)